src/poetry/core/utils/_compat.py

```diff
@@ -3,9 +3,6 @@
 from typing import List
 
 
-PY36 = sys.version_info >= (3, 6)
-PY37 = sys.version_info >= (3, 7)
-
 WINDOWS = sys.platform == "win32"
 
 
```

Comment:
is the following guard still required?

```python
try:
    FileNotFoundError
except NameError:
    FileNotFoundError = IOError  # noqa
```
?

Reply:
Good catch, we're not using it